### DISCUSSION TITLE
Callout icons: color o highlight

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1172] [Q&A]
Hola, Tenemos una duda de si el callout debe llevar el icono en neutralHigh o en brand por defecto.
En Brand Factory se muestra en brand, y en las specs salen en neutralHigh.
¿Mística recomienda usar colores en los iconos de callout o recomienda usar solo neutralHighlight?

**Comment by yceballost** [comment]
Hello!

El uso del icono en el callout realmente es libre, está más cerca de una decisión de producto que de Mística. De hecho, para la marca Movistar tendría más sentido que se usaran en neutralHigh porque buscan desde marca reducir el peso de los iconos y los colores.

Personalmente, creo que mantendía el color unicamente cuando sean estados o se quiera comunicar marca con el mensaje del callout.

Si te parece bien y para no generar esa duda desde Mística voy a cambiar el color del icono en BF para que use neutralHigh

**Comment by MontseTelefonica** [comment]
Perfecto! Gracias!

**Reply by yceballost** [reply]
cierro!
